joint_trajectory_controller/src/joint_trajectory_controller.cpp

```diff
@@ -350,6 +350,12 @@ controller_interface::return_type JointTrajectoryController::update(
       }
     }
   }
+  else
+  {
+    // TODO(peterdavidfagan) temporary solution to keep the initial position
+    // before the first trajectory goal arrives
+    set_hold_position();
```

Comment:
Thanks for the feedback @destogl I will review the current set of commits based on the above information. It may be tomorrow before I can make updates due to an item I am finishing this evening.